.github/workflows/review_status.yml

```diff
@@ -0,0 +1,124 @@
+name: "Synchronize PR status with development board"
+
+on:
+  pull_request_review:
+    types: [submitted, edited, dismissed]
```

Comment:
IMO this won't work as the GITHUB_TOKEN will not have sufficient privileges.

Reply:
I'm not sure why the CI run on the PR branch picked up GH_TOKEN as ${{ github.token }}, but the yaml file pretty clearly sets GH_TOKEN as follows:

```
env:
  GH_TOKEN: ${{ secrets.PROJECT_SECRET }}
```

Where secrets.PROJECT_SECRET is a github token with sufficient privileges to do this work